### ISSUE TITLE
Add a params ad docstring to run function in main.star

### ISSUE THREAD

**Issue by tedim52** [kurtosis-tech/ethereum-package#364]
The `main.star` should contain a doc string that specifies exactly what the possible input args are, which ones are required and which ones are default, and what their descriptions are types are. This will make it easier for users to get onboarded to the ethereum package as they'll know what's parametrizable via a json/yaml file without having to go to `package_io` or go off solely example inputs, briding the [author consumer divide](https://www.kurtosis.com/blog/author-consumer-divide). 

A good example to go off of is the [hyperlane package](https://github.com/kurtosis-tech/hyperlane-package/blob/52c44b50d91fc2d428ba6e86905c19efa6510c76/main.star#L6).

Doing this will also make the package render with a nice form input in Kurtosis Cloud like by parsing docstring

<img width="1050" alt="Screen Shot 2023-11-19 at 3 20 07 AM" src="https://github.com/kurtosis-tech/ethereum-package/assets/46531991/d3877d43-c57b-4e97-b841-925533d716e1">

Whereas the ethereum package one looks like this:
<img width="1046" alt="Screen Shot 2023-11-19 at 3 21 01 AM" src="https://github.com/kurtosis-tech/ethereum-package/assets/46531991/09baed5c-742a-4ad4-9aa1-8a381b03b926">

**Comment by barnabasbusa** [comment]
I think if we go down this road, this should be hidden under a group that is collapsable. 

And with the default params filled out. And with the option to override all with a single yaml config.

**Comment by tedim52** [comment]
Yep. This is all possible with the docstring! Can specify a default that can be overriden very easily with the yaml configs that already exist. I'll create a PR and request ya to demonstrate how it works.

**Comment by mieubrisse** [comment]
Default params for primitive values are actually already implemented in the newest version of Kurtosis:
<img width="1020" alt="Screen Shot 2023-11-21 at 21 45 12" src="https://github.com/kurtosis-tech/ethereum-package/assets/1142185/15900a49-4d2e-48f2-9af3-98163a4d48aa">

The option to override all with a single YAML config is definitely something we'll have in the near term (though we don't have it yet).

Collapsible groups - what did you have in mind @barnabasbusa ?

**Comment by barnabasbusa** [comment]
ethereum-package will have tens of different config parameters. 

If you have all of them "open" in a non collapsed manner, its gonna take up the whole page. 

Would be good to separate them into parts, such as participant group, network config, other configs, additional-tooling.

**Comment by mieubrisse** [comment]
This is possible simply by making them separate parameters to the `run` function, e.g.:
```python
def run(plan, participants=[], additional_tools=[], etc.):
```
This will also allow each of them to have separate documentation, a la: https://github.com/kurtosis-tech/hyperlane-package/blob/main/main.star#L24
